src/javascript/typescript/codeGeneration.ts

```diff
@@ -164,19 +163,18 @@ export class TypescriptAPIGenerator extends TypescriptGenerator {
     const properties = this.getPropertiesForVariant(variant);
 
     const exportedTypeAlias = this.exportDeclaration(
-      this.typeAliasObject(operationName, properties)
+      this.interface(operationName, properties)
     );
 
     this.printer.enqueue(exportedTypeAlias);
     this.scopeStackPop();
   }
 
-  public typeAliasesForFragment(fragment: Fragment) {
+  public typesForFragment(fragment: Fragment) {
```

Comment:
thinking about this now, we might as well call this `interfacesForFragment` since that's actually whats being generated

Reply:
You're right 👍